testci/install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+# This script is meant to be called by the "install" step defined in
+# .travis.yml. See http://docs.travis-ci.com/ for more details.
+# The behavior of the script is controlled by environment variabled defined
+# in the .travis.yml in the top level folder of the project.
+#
+# This script is adapted from a similar script from the scikit-learn repository.
+# Taken from: https://github.com/aabadie/nilearn/blob/circle_ci_anaconda/continuous_integration/install.sh
+#
+# License: 3-clause BSD
+
+set -e
+
+# Fix the compilers to workaround avoid having the Python 3.4 build
+# lookup for g++44 unexpectedly.
+export CC=gcc
+export CXX=g++
+
+create_new_venv() {
+    # At the time of writing numpy 1.9.1 is included in the travis
+    # virtualenv but we want to be in control of the numpy version
+    # we are using for example through apt-get install
+    deactivate
+    virtualenv --system-site-packages testvenv
+    source testvenv/bin/activate
+    pip install nose
+}
+
+print_conda_requirements() {
+    # Echo a conda requirement string for example
+    # "pip nose python='2.7.3 scikit-learn=*". It has a hardcoded
+    # list of possible packages to install and looks at _VERSION
+    # environment variables to know whether to install a given package and
+    # if yes which version to install. For example:
+    #   - for numpy, NUMPY_VERSION is used
+    #   - for scikit-learn, SCIKIT_LEARN_VERSION is used
+    TO_INSTALL_ALWAYS="pip nose"
```

Comment:
Do we want to change the list of these packages? For example we do not use `nose`, we use `pytest`.